.circleci/soltest.ps1

```diff
@@ -0,0 +1,4 @@
+cd "$PSScriptRoot\.."
+mkdir test_results
+.\build\test\Release\soltest.exe --color_output=no --show_progress=yes --logger=JUNIT,error,test_results/result.xml -- --no-smt
+.\build\test\Release\soltest.exe --color_output=no --show_progress=yes --logger=JUNIT,error,test_results/result_opt.xml -- --optimize --no-smt
```

Comment:
Do you agree that it's better to run at least those two? I originally added the second one thinking "well it takes less than a minute, so why not", but the optimized run takes longer of course - but together it's still only a tad more than 5 minutes.

Reply:
Makes sense

Reply:
Sounds good!